{Release} Upgrade to Azure CLI 2.6.0 (#13542)
Azure/azure-cli: src/azure-cli/HISTORY.rst

Release History
===============

2.6.0
++++++

**ACR**

* Add default timeout of 5 minutes for any requests to ACR (#13349)
* Support disable public network access (#13347)
* `az acr token create`: expose --days argument (#13392)
* `az acr import`: accept --source argument values which contain login in server name through client end correction (#13392)

**ACS**

> * bug fix: remove fields cleanup for fields that no longer exist (#13315)

Reviewer: ```suggestion
* Bug fix: remove fields cleanup for fields that no longer exist (#13315)
```

> * bug fix: remove fields cleanup for fields that no longer exist (#13315)

Reviewer 2: This a bad description/history. 

> Fix #xxx: `az xxx`: xxx

**AKS**

* Update uptime-sla command help context (#13300)
* Remove range check for updating min count for autoscaler (#13215)
* Fix that cli doe not fail when user only specifies Windows password (#13418)

**AMS**

> * Change: az ams transform create: Added ability to create a transform with a FaceDetector preset (#13260)

Reviewer: ```suggestion
* Change: az ams transform create: Add ability to create a transform with a FaceDetector preset (#13260)
```

> * Change: az ams transform create: Added ability to create a transform with a FaceDetector preset (#13260)
> * CHange: az ams content-key-policy create : Added ability to create a FairPlay content key policy with an offline rental configuration (#13260)

Reviewer 3: ```suggestion
* `az ams transform create`: Add ability to create a transform with a FaceDetector preset (#13260)
* `az ams content-key-policy create` : Add ability to create a FairPlay content key policy with an offline rental configuration (#13260)
```

> * CHange: az ams content-key-policy create : Added ability to create a FairPlay content key policy with an offline rental configuration (#13260)

Reviewer: ```suggestion
* Change: az ams content-key-policy create : Add ability to create a FairPlay content key policy with an offline rental configuration (#13260)
```

**AppConfig**

> * Bug fix for list key values with fields (#13326)

Reviewer 2: This a bad description/history.

**ARM**

* `az deployment`: Fix the problem that the templateLink will not be returned when deploying or validating template-uri (#13317)
* `az deployment`: Fix the problem that deployment/validate does not support specially encoded character (#13137)
* `az deployment sub/group what-if`: Fix array alignment and error handling (#13295)
* `az deployment operation`: Modify the deprecate information (#13129)

**ARO**

* Add examples to az aro create, list, list-credentials, show, delete (#13403)
* Add generate_random_id function (#13482)

**Backup**

* Allow FriendlyName in enable protection for AzureFileShare command (#13268)
* Fix in IaasVM restore-disks Command (#13348)
* Add "MAB" BackupManagementType to item list command (#13449)
* Add support for retrying policy update for failed items. (#13432)
* Add Resume Protection functionality for Azure Virtual Machine (#13396)
* Add support to specify ResourceGroup for storing instantRP during Create or Modify Policy (#13376)

**CI**

* Support flake8 3.8.0 (#13454)

**Compute**

* New command az vm auto-shutdown (#13199)
* `az vm list-skus`: Update --zone behavior, return all type skus now (#13470)

> 

Reviewer 4: ```suggestion

2.6.0
++++++
* Update local context on/off status to global user level (#13277)

```
